tests/unit_tests/main_test.cpp

```diff
@@ -102,9 +102,9 @@ int main(int argc, char** argv) {
             for (auto dev : plat_devs) {
                 try {
                     /* Do not test for OpenCL backend on GPU */
-                    if (dev.is_gpu() && plat.get_info<sycl::info::platform::name>().find(
-                                            "OpenCL") != std::string::npos)
-                        continue;
+                    // if (dev.is_gpu() && plat.get_info<sycl::info::platform::name>().find(
+                    //                         "OpenCL") != std::string::npos)
+                    //     continue;
```

Comment:
I think you didn't mean to push that.

Reply:
I did not - thank you for spotting this. Reverted in [d0d10fe](https://github.com/oneapi-src/oneMKL/pull/411/commits/d0d10febbaf898e36bdd05f8579ef1a5338e0020).